test/truffle/integration/js.sh

```diff
@@ -1,6 +0,0 @@
-#!/usr/bin/env bash
-
-source test/truffle/common.sh.inc
-
-jt ruby test/truffle/integration/js/eval.rb
-jt ruby test/truffle/integration/js/inline-exported.rb
```

Comment:
These are run in GraalVM, no?

Reply:
I don't think so? Do we ever do a monolithic build now?

But anyway, I don't think it's our job to test language integration. We now have the better TCK. These are just badly written noise now.

Reply:
I think test/truffle/integration/js/transparent.rb would still be nice to test.
It could move, but where would we add it?
The rest seems mostly polyglot eval so indeed not so interesting.

Reply:
Ah right yes transparent is something else. I'll restore that.